File: URI/1009.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>

	char nome[20];
	double salario, montante,total;

int main(){
	scanf("%s",&nome[20]);
	scanf("%lf %lf",&salario,&montante);
	total=salario+(montante/100)*15;
	printf("TOTAL = R$ %.2lf\n",total);
	
	return 0;
}

File: URI/Rust/1009.rs
use std::io;

fn main() {
	let mut nome = String::new();
	let mut salario = String::new();
	let mut vendas = String::new();

	io::stdin()
		.read_line(&mut nome)
		.expect("Falha 1");
	io::stdin()
		.read_line(&mut salario)
		.expect("Falha 2");
	io::stdin()
		.read_line(&mut vendas)
		.expect("Falha 3");
	
	let salario: f64 = salario.trim().parse().expect("Falha 5");
	let vendas : f64 = vendas.trim().parse().expect("Falha 6");

	let total = salario + vendas * 0.15;

	println!("TOTAL = R$ {:.2}",total);
}